src/cryptonote_config.h

```diff
@@ -257,6 +257,11 @@ namespace config
   inline constexpr std::string_view HASH_KEY_CLSAG_AGG_0 = "CLSAG_agg_0"sv;
   inline constexpr std::string_view HASH_KEY_CLSAG_AGG_1 = "CLSAG_agg_1"sv;
 
+  //Batching SN Rewards
+  inline constexpr uint64_t BATCHING_INTERVAL = 6;
```

Comment:
TODO more testing to decide what the optimal length for this batching interval is. Preliminary python scripts were indicating that 200 possibly is a good length for a 40% reduction in block outputs. 2000 will get us an 80% reduction however I have some concerns about coinbase payments having too many outputs in a single block if we do that high.